### ISSUE TITLE
Add command to run a command in specific virtualenv

### ISSUE THREAD

**Issue by tomplex** [tomplex/virtualb#3]
Not sure what it should be called yet.

**Comment by awburgess** [comment]
Vote as second between #1 and #4

**Comment by awburgess** [comment]
Perhaps ```remote```?

**Comment by tomplex** [comment]
oh, I like that. I was actually trying to decide what the syntax should be. `vb remote [env] [cmd]`?

**Comment by tomplex** [comment]
Was also considering `exec` or `cmd` for the command name. But in that case I think it would make more sense to change the argument order; i.e. `vb exec [cmd] [env]`

**Comment by tomplex** [comment]
What is the expected behavior here? I just finished implementing something for this and realized that there are multiple ways to interpret what this ticket is getting after.

1. Literally execute an arbitrary command and have it be interpreted as if the command was run in bash with that environment active
2. Pass an arbitrary command to that environment's Python executable, i.e. `vb exec some_env  '-c "import sys; print(sys.path[2])"'` or `vb exec some_env some_file/to/run.py`
3. Assume that the command is actually Python that we want to execute; essentially force the `-c` flag (I don't think this is the expected behavior).

I implemented this assuming option 2, but I wanted to check in and see what you thought, @awburgess.
